build.gradle.kts

```diff
@@ -94,15 +96,20 @@ dependencies {
     implementation("org.apache.tinkerpop:gremlin-core:3.5.1")
     implementation("org.jgrapht:jgrapht-core:1.5.1")
 
+    implementation("com.beust:klaxon:5.5")
+
+    implementation("io.github.oshai:kotlin-logging-jvm:5.0.0")
+    implementation(group = "ch.qos.logback", name = "logback-classic", version = "1.2.6")
     implementation("it.unimi.dsi:fastutil:8.5.12")
     implementation("org.lz4:lz4-java:1.8.0")
 
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.0")
 
-    val kotestVersion = "4.2.6"
-    listOf("runner-junit5", "assertions-core", "property").forEach {
+    val kotestVersion = "5.6.2"
```

Comment:
Would this change be picked up if you synced with master?  That should happen before each pull request.